package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-create-ui",
-  "version": "3.5.0",
+  "version": "3.5.1",
```

Comment:
Main branch is already at 3.5.1. Please change this to 3.5.2 (or ignore if it doesn't matter to you, this time, to track the version in the deployed app).